src/ResponseValidator.ts

```diff
@@ -209,26 +203,21 @@ export class ResponseValidator {
     }
 
     protected _filterProtocolClaims(claims: UserProfile): UserProfile {
-        this._logger.debug("_filterProtocolClaims, incoming claims:", claims);
```

Comment:
at some point we should log the incoming claims, if not here then here
`logger.debug("response is OIDC, processing claims", response.profile);`?